packages/components/src/pseudolocalize.js

```diff
@@ -0,0 +1,95 @@
+// modified from https://searchfox.org/mozilla-central/source/intl/l10n/L10nRegistry.jsm
+// removed option for 'flipped' to test RTL
+
+const ACCENTED_MAP = {
```

Comment:
Does this end up in live code? Should we ensure it doesn't?

Reply:
That would be at the discretion of projects using this package to determine how they set up the string transformation settings to use pseudolocalization, and whether it is accessible in live environments or not.